code-reviews/recipes/Terraform.md

```diff
@@ -53,10 +58,16 @@ echo -e "\n\n>>> Terraform validate"
 terraform validate
 ```
 
-> TODO: Add example of how to use this in an AzDO build validation pipeline
-
-## Code Review Checklist
-
+### Code Review Checklist
```

Comment:
General comment around the check list... for consistency with the check lists in the other recipes. please use 

- [ ] Check list item

and make the check list items questions... 
example 

- [ ] Is the Terraform project configured using Azure Storage as a remote state backend?
- [ ] Is the remote state backend storage account key stored in the Key vault?
- [ ] Is all Terraform code isolated in a dedicated folder?
...

Reply:
Updated based on the feedback.